src/Controller/Api/UserController.php

```diff
@@ -68,7 +71,7 @@ public function __construct(UserManagerInterface $userManager, GroupManagerInter
      *  output={"class"="Sonata\DatagridBundle\Pager\PagerInterface", "groups"={"sonata_api_read"}}
      * )
      *
-     * @Get("/users", name="get_users")
+     * @Get("/users.{_format}", name="get_users")
```

Comment:
```suggestion
     * @Get("/users.{_format}", name="get_users", requirements={"_format": "json|xml|html"})
```
I check it and compare with [demo](https://demo.sonata-project.org/). I think we should add requirements for it too. I don't know what when people set rules for it or move some format. WDYT? @phansys

Reply:
I've added the requirements globally in each controller via `@Route` annotation in order to share the definition for all the routes in that scope.